cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -662,6 +662,10 @@ spec:
               - passive
               - aggressive
               - force
+            enableRBDStats:
```

Comment:
Any changes to the schema are needed in two more places:
- Helm chart: cluster/charts/rook-ceph/templates/resources.yaml
- Integration tests: tests/framework/installer/ceph_manifests.go

Reply:
ack.